src/test/java/com/wikia/webdriver/testcases/discussions/PollsTests.java

```diff
@@ -138,7 +138,7 @@ public void anonUserCanNotVoteInPollOnMobile() {
         manageSignInModal(poll);
     }
 
-    @Test(dependsOnMethods = {"userCanCreatePostWithSimplePollOnMobile"}, groups = "discussions-polls")
+    @Test()//dependsOnMethods = {"userCanCreatePostWithSimplePollOnMobile"}, groups = "discussions-polls")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)